torbrowser/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: torbrowser_web_1
+      APP_PORT: 5800
+      PROXY_AUTH_ADD: "false"
+  web:
+    image: ghcr.io/lkshrk/docker-tor-browser:14.0.3@sha256:9d854697d4afa21a6e6e8fb98d0cf2a82776330c95d9b5bb03f3665096b3f1da
+    restart: on-failure
+    environment:
+      - TZ=Europe/Berlin
```

Comment:
We should remove the timezone environment variable or set it to utc, as the app can be installed by anyone from around the globe
```suggestion
```

Reply:
I decided to remove the line.